content/docs/installation/continuous-deployment-and-gitops.md

```diff
@@ -112,3 +112,141 @@ Check the cert-manager logs for warnings and errors:
 ```bash
 kubectl logs -n cert-manager -l app.kubernetes.io/instance=cert-manager --prefix --all-containers
 ```
+
+
+## Using ArgoCD
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+### Pre-requisites
+Ensure the following are in place before proceeding:
+- A Kubernetes cluster
+- ArgoCD deployed on the Kubernetes cluster: [installation guide](https://argo-cd.readthedocs.io/en/stable/getting_started/)
+- Optional: A GitOps repository connected with ArgoCD: [setup guide](https://argo-cd.readthedocs.io/en/stable/user-guide/private-repositories/)
+
+### Setting up cert-manager
+1. Create an ArgoCD Application manifest file with the provided configuration to set up cert-manager.
+
+    ```yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: cert-manager
+      namespace: argocd
+      annotations:
+        argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+      finalizers:
+        - resources-finalizer.argocd.argoproj.io
+    spec:
+      destination:
+        namespace: cert-manager
+        server: https://kubernetes.default.svc
+      project: default
+      source:
+        chart: cert-manager
+        repoURL: https://charts.jetstack.io
+        targetRevision: 1.10.1
+        helm:
+          values: |
+            installCRDs: true
+      syncPolicy:
+        automated:
+          prune: true
+          selfHeal: true
```

Comment:
Add YAML comments above these fields explaining why these options are needed.

Reply:
These are part of the (Argo) Application resource. Linking to the official documentation for this resource in the Argo documentation should be enough.
I think we want people to go read the official docs vs just copying and pasting from this docs.